### ISSUE TITLE
Mention coordinated omission in arrival-rate docs

### ISSUE THREAD

**Issue by na--** [k6io/docs#231]
Prompted by https://community.k6.io/t/is-k6-safe-from-the-coordinated-omission-problem/1484

We should explicitly mention and explain coordinated omission in https://k6.io/docs/using-k6/scenarios/arrival-rate. It is the popular term and some links to Gil Tene's lectures won't hurt.

btw having a "best practices" page like JMeter is probably also a good idea... https://jmeter.apache.org/usermanual/best-practices.html

**Comment by simskij** [comment]
Definitely! A best practices page has been on the agenda for quite a while.
